theano/compat/__init__.py

```diff
@@ -6,15 +6,16 @@
 from six import PY3, b, BytesIO, next
 from six.moves import configparser
 from six.moves import reload_module as reload
+from collections import OrderedDict
 try:
-    from collections.abc import (OrderedDict, MutableMapping as DictMixin,
-                                 Callable)
+    from collections.abc import Callable, Iterable, Mapping, ValuesView
+    from collections.abc import MutableMapping as DictMixin
```

Comment:
Why change the style syntax? I would just leave it as is with the parentheses.

Reply:
I can do that. The code was changed twice in the row (in the first commit, the import was shorter, in the second it turned out to be longer again, so I just did what I consider more readable).

Reply:
Rebased.